### PR TITLE
Add Create/Delete Project API

### PR DESCRIPTION
**Pull request by devholic**
This PR adds Create/Delete Project API for letting users to manage Project without `kubectl`.

### CODE REVIEW

File: internal/api/handler/create_project_v1alpha1.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+func CreateProjectV1Alpha1(
+	kc client.Client,
+) CreateProjectV1Alpha1Func {
+	return func(
+		ctx context.Context,
+		req *connect.Request[svcv1alpha1.CreateProjectRequest],
+	) (*connect.Response[svcv1alpha1.CreateProjectResponse], error) {
+		name := strings.TrimSpace(req.Msg.GetName())
+		if name == "" {
+			return nil, connect.NewError(connect.CodeInvalidArgument, errors.New("name should be empty"))
```
Comment (krancour) [inline internal/api/handler/create_project_v1alpha1.go:32]: Should this read "name should NOT be empty"?

File: internal/api/handler/delete_project_v1alpha1.go [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+		ctx context.Context,
+		req *connect.Request[svcv1alpha1.DeleteProjectRequest],
+	) (*connect.Response[svcv1alpha1.DeleteProjectResponse], error) {
+		name := strings.TrimSpace(req.Msg.GetName())
+		if name == "" {
+			return nil, connect.NewError(connect.CodeInvalidArgument, errors.New("name should be empty"))
+		}
+
+		var ns corev1.Namespace
+		if err := kc.Get(ctx, client.ObjectKey{Name: name}, &ns); err != nil {
```
Comment (krancour) [inline internal/api/handler/delete_project_v1alpha1.go:35]: Are there delete options that can take labels into account? If so, you might be able to get by without the get.

**Review by krancour — APPROVED** [review]
I have a few minor questions and comments, but none of them are blocking my approval.

LGTM!

File: internal/api/handler/create_project_v1alpha1.go [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+) (*connect.Response[svcv1alpha1.CreateProjectResponse], error)
+
+func CreateProjectV1Alpha1(
+	kc client.Client,
+) CreateProjectV1Alpha1Func {
+	return func(
+		ctx context.Context,
+		req *connect.Request[svcv1alpha1.CreateProjectRequest],
+	) (*connect.Response[svcv1alpha1.CreateProjectResponse], error) {
+		name := strings.TrimSpace(req.Msg.GetName())
```
[the remark this answers is not in the document]
Comment (gdsoumya) [reply internal/api/handler/create_project_v1alpha1.go:30]: Yea not a blocker we can always add it later

**Review by rbreeze — APPROVED** [review]
LGTM

**Pull request merged**